app/models/message.rb

```diff
@@ -46,6 +46,9 @@ class Message < ApplicationRecord
 
   after_update_commit ->(message) { EventBus.publish(:message_changed, message) }
   after_destroy_commit ->(message) { EventBus.publish(:message_destroyed, message) }
+  after_create_commit do |message|
+    broadcast_render_later_to message.thread, partial: "messages/new_message_alert", locals: { message: message }
```

Comment:
Do ktorej fronty sa toto zaradi? Beriem to tak, ze by to malo mat dost vysoku prioritu kedze to je userfacing.